plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/web/aspects/Highlight.java

```diff
@@ -53,7 +53,7 @@ private void highlightElementOn(WebElement webElement) {
     private void highlightElementOff(WebElement webElement, String originalStyle) {
         originalStyle = (originalStyle == null) ? "" : originalStyle;
         try {
-            JavascriptExecutor js = (JavascriptExecutor) PageContext.getCurrentPage().getDriver();
+            JavascriptExecutor js = (JavascriptExecutor) Environment.getDriverService().getDriver();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast to "JavascriptExecutor". ([squid:S1905](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1905))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=95&issues=AWZXmwwYvOM6zLh9xWGq&open=AWZXmwwYvOM6zLh9xWGq)

[ISSUE_KEY]: AWZXmwwYvOM6zLh9xWGq